blob/src/main/java/com/bazaarvoice/emodb/blob/core/BlobStoreProviderProxy.java

```diff
@@ -123,7 +122,7 @@ public Blob get(String table, String blobId, @Nullable RangeSpecification rangeS
     }
 
     @Override
-    public void put(String table, String blobId, InputSupplier<? extends InputStream> in, Map<String, String> attributes)
+    public void put(String table, String blobId, java.util.function.Supplier<? extends InputStream> in, Map<String, String> attributes)
```

Comment:
Shouldn't we import it?

Reply:
we cannot do that because of `import com.google.common.base.Supplier;` row